src/iocore/net/SSLUtils.cc

```diff
@@ -1117,14 +1119,14 @@ void
 SSLMultiCertConfigLoader::_set_handshake_callbacks(SSL_CTX *ctx)
 {
   // Make sure the callbacks are set
-#ifndef OPENSSL_IS_BORINGSSL
+#if !HAVE_SSL_CTX_SET_SELECT_CERTIFICATE_CB // For OpenSSL < 1.1.1, we should be able to remove this
```

Comment:
Don't we already require OpenSSL >= 1.1.1?

Reply:
Yes, we do. I didn't want to change any logic on this PR so I just left the comment.